palisade-service/src/test/java/uk/gov/gchq/palisade/service/palisade/service/SimplePalisadeServiceTest.java

```diff
@@ -0,0 +1,196 @@
+        when(userService.getUser(any(GetUserRequest.class))).thenReturn(user);
+        when(resourceService.getResourcesById(any(GetResourcesByIdRequest.class))).thenReturn(resources);
+        when(policyService.getPolicy(any(GetPolicyRequest.class))).thenReturn(multiPolicy);
+
+        RegisterDataRequest request = new RegisterDataRequest()
+                .userId(new UserId().id("Bob"))
+                .context(new Context().purpose("Testing"))
+                .resourceId("/path/to/new/bob_file.txt");
+
+        //When
+        DataRequestResponse actualResponse = service.registerDataRequest(request).toCompletableFuture().join();
+        actualResponse.originalRequestId(request.getId());
+
+        //Then
+        assertEquals(expectedResponse.getResources(), actualResponse.getResources());
+    }
+
+    @Test(expected = CompletionException.class)
+    public void getDataRequestConfigFromEmptyCacheTest() {
+        LOGGER.info("Expected config: {}", expectedConfig);
+        //Given
+        GetDataRequestConfig requestConfig = new GetDataRequestConfig();
+        requestConfig.requestId(new RequestId().id("requestId"));
+        requestConfig.resource(new FileResource().id("resourceId"));
+        LOGGER.info("Get Data Request Config: {}", requestConfig);
+
+        //When
+        CompletableFuture<DataRequestConfig> cacheConfig = service.getDataRequestConfig(requestConfig);
+        cacheConfig.toCompletableFuture().join();
+    }
+
+    private void createExpectedDataConfig() {
+        User user = new User();
+        Context context = new Context();
+        Map<LeafResource, Rules> ruleMap = new HashMap<>();
+        FileResource resource = new FileResource();
+        Rules rules = new Rules();
+        Set<String> roles = new HashSet<>();
+        Set<String> auth = new HashSet<>();
+
+        user.setUserId(new UserId().id("userId"));
+        roles.add("roles");
+        user.setRoles(roles);
+        auth.add("auth");
+        user.setAuths(auth);
+        context.purpose("purpose");
+        ruleMap.put(resource, rules);
+        expectedConfig.setUser(user);
+        expectedConfig.setContext(context);
+        expectedConfig.setRules(ruleMap);
+    }
+
+    private void mockOtherServices() {
+        auditService = Mockito.mock(AuditService.class);
+        policyService = Mockito.mock(PolicyService.class);
+        resourceService = Mockito.mock(ResourceService.class);
+        userService = Mockito.mock(UserService.class);
+        aggregationService = Mockito.mock(ResultAggregationService.class);
+    }
+
+    private void setupCacheService() {
+        final BackingStore store = Mockito.mock(BackingStore.class);
+        serviceState.put("cache.svc.store", null);
+        serviceState.put("cache.svc.max.ttl", Duration.of(5, ChronoUnit.MINUTES).toString());
+        cacheService = new SimpleCacheService();
+        cacheService.recordCurrentConfigTo(serviceState);
```

Comment:
This relates to the old configuration service that is no longer needed in SpringBoot, The recordCurrentConfigTo method can be deleted entirely. I suspect that once that is done, this test can be binned.

Reply:
This refactor will be done in a separate PR.